teddy.js

```diff
@@ -382,11 +365,44 @@
 
       // compile template if necessary
       if (!teddy.templates[template] || teddy.params.compileAtEveryRender) {
-        renderedTemplate = teddy.compile(template)
+        renderedTemplate = teddy.compile(template, dontParse)
```

Comment:
still gotta remove the unneeded dontParse argument from all compile calls

Reply:
I removed it everywhere.